README.md

```diff
@@ -13,12 +13,18 @@ Clone this repository:
 $ git clone git@github.com:Asquera/wheelmap-devenv.git
 ```
 
+First run
+
+```
+$ vagrant init
```

Comment:
This is not necessary. `vagrant init` generates a completely new `Vagrantfile` but since we already have one this can be omitted.